gocourse05/camera/day_test.go

```diff
@@ -0,0 +1,58 @@
+package camera
+
+import (
+	"testing"
+)
+
+func TestDayCameraRetrieveData(t *testing.T) {
+	t.Run("Found", func(t *testing.T) {
+		c := NewDayCamera(1, "Canon", []Data{NewDayCameraData(1, "Коза", "Побігла вліво")})
+
+		data, err := c.RetrieveData()
+		if err != nil {
+			t.Errorf("RetrieveData() should not have error: %s", err)
+		}
+
+		if len(data) == 0 {
+			t.Errorf("RetrieveData() should not return empty data.")
+		}
+
+		for _, d := range data {
+			if d.Animal != "Коза" || d.Movement != "Побігла вліво" {
+				t.Errorf("RetrieveData() = %+v", d)
+			}
+		}
```

Comment:
Здається, простіше ось так:
```suggestion
		got := data[0]
		want := Data{ID: 1, Animal: "Коза", Movement: "Побігла вліво"}
		if got != want {
			t.Errorf("Unexpected data: got=%+v, want=%+v", got, want)
		}
```

І відповідає
https://go.dev/wiki/CodeReviewComments#useful-test-failures